godot-core/src/builtin/collections/array.rs

```diff
@@ -285,14 +285,15 @@ impl<T: ArrayElement> Array<T> {
         self.as_inner().is_empty()
     }
 
-    /// Returns a 32-bit integer hash value representing the array and its contents.
+    crate::declare_hash_u32_method! {/// Returns a 32-bit integer hash value representing the array and its contents.
```

Comment:
```suggestion
    crate::declare_hash_u32_method! {
        /// Returns a 32-bit integer hash value representing the array and its contents.
```
also indentation of subsequent lines.